libsolidity/inlineasm/AsmCodeGen.h

```diff
@@ -47,7 +48,8 @@ class CodeGenerator
 		AsmAnalysisInfo& _analysisInfo,
 		eth::Assembly& _assembly,
 		yul::ExternalIdentifierAccess const& _identifierAccess = yul::ExternalIdentifierAccess(),
-		bool _useNamedLabelsForFunctions = false
+		bool _useNamedLabelsForFunctions = false,
+		AssemblyStack::Language _language = AssemblyStack::Language::StrictAssembly
```

Comment:
Could you move that right after `_assembly`? Also I think it should not have a default value.